packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -147,19 +147,21 @@ class ConnectionStateCatchup extends ConnectionStateHandlerPassThrough {
         switch (value) {
             case ConnectionState.Connected:
                 assert(this._connectionState === ConnectionState.CatchingUp, 0x3e1 /* connectivity transitions */);
```

Comment:
It's a little confusing to consider how this layer's connectionState does/should align with the `oldState` arg. Is that parameter even necessary?

Reply:
This layer does not touch `oldState`, but it used to propagate to next consumer / adapter in case it's a pass-through case.